docs/src/modules/developing/pages/development-process-js.adoc

```diff
@@ -79,6 +78,6 @@ xref:javascript:run-locally.adoc[running them locally] before deploying to Akka
 
 After testing locally, deploy your service to Akka Serverless using the CLI or the Console. The following pages provide information about deployment:
 
-* https://developer.lightbend.com/docs/akka-serverless/getting-started/projects.html[Working with Akka Serverless deployment projects]
-* https://developer.lightbend.com/docs/akka-serverless/deploying/deploying.html[Deploying a packaged service]
-* https://developer.lightbend.com/docs/akka-serverless/tutorial/index.html[Examples of how to deploy]
+* https://developer.lightbend.com/docs/akka-serverless/projects/index.html[Working with Akka Serverless deployment projects]
+* https://developer.lightbend.com/docs/akka-serverless/services/deploy-service.html#_deploy[Deploying a packaged service]
+* https://developer.lightbend.com/docs/akka-serverless/quickstart/cr-value-entity-javascript.html#_package_and_deploy_your_service[Example of how to package and deploy a service]
```

Comment:
@retgits subjective decision here, could you please confirm?

Reply:
I'd choose these too, also could you call it `Working with Akka Serverless projects` 😇

Reply:
Done